src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -18,7 +18,7 @@ import { AUTHENTICATOR_KEY } from '../RemediationConstants';
  * @param {AuthenticatorOption[]} options
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
-const createOVOptions = (options = []) => {
+export const createOVOptions = (options = []) => {
```

Comment:
Since you're changing this anyway, can we change it to a function instead of a const. -
```
export function createOVOptions(options = []) {
```

Reply:
It might be in our best interest to change other exports here too. There's no need for them to be consts as they all are functions. And exporting individual functions supports tree-shaking better (if we were to tree-shake soon.)